CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.6.0 / 2018-xx-xx
+
+* Switch to Alpine for smaller images
+* Switch to Quagga or Zebra deamon
```

Comment:
Might be a good idea to add a note regarding zapi version change from 4 to 3. Last time I had trouble with version < 4 with gobgp.

Reply:
Zebra protocol version 4 seems to specific to the FRR version. The "original" Zebra seem to support only 0 and 1, while Quagga adds version 3.

The only reason I changed this to Quagga is that there a Alpine packages for Quagga. FRR has been working on Alpine packages as well, but they are somewhat WIP (https://github.com/FRRouting/frr/commits/master/alpine).

It might make sense investigate if the FRR apline packages can be integrated instead of the Quagga ones.